src/content/community/Community-Resources.md

```diff
@@ -3,122 +3,38 @@ title: Community Resources
 layout: docs
 category: Community
 permalink: /community/
-next: /community/upcoming-events/
-sublinks: Blogs,Videos,Books,More Resources
+next: /community/users/
+sublinks: Official channels,GraphQL Trademark,Code of conduct
 ---
 
-## Stack Overflow
+The GraphQL community is worldwide, and includes developers, users, supporters, and fans from around the globe. You are welcome to join us!
 
-Many members of the community use Stack Overflow to ask and answer questions. [Read through the existing questions tagged with **graphql** or ask your own!](http://stackoverflow.com/questions/tagged/graphql)
+If you're new to the community, here are a few things to get you started.
 
-## Facebook Group
+## Official channels
 
-Join the [GraphQL Facebook Group](https://www.facebook.com/groups/graphql.community/) sharing and discovering new content. The GraphQL Facebook group is the preferred venue for announcements and broader discussion.
+The GraphQL community is active and has created a tremendous amount of valuable content, but we maintain a number of primary collaboration channels for the project.
 
-## Twitter
+Please follow these channels for updates and information:
 
-Use the [**#graphql** hashtag](https://twitter.com/search?q=%23GraphQL&src=tyah) on Twitter to join the conversation.
+* **Twitter:** [@graphql](https://twitter.com/graphql)
+* **GitHub:** [github.com/graphql](https://github.com/graphql)
+* **Slack:** [slack.graphql.org](https://slack-invite.graphql.org)
```

Comment:
It's odd that these link to different places. I feel like it's deliberate, but it's also unexpected, so maybe we should split them up:

```suggestion
* **Slack:** [slack.graphql.org](https://slack.graphql.org) ([get an invite](https://slack-invite.graphql.org))
```

Reply:
Works for me!